assets/tests/components/mapPage/mapDisplay.test.tsx

```diff
@@ -860,4 +861,29 @@ describe("<MapDisplay />", () => {
       expect(mockSetSelection).not.toHaveBeenCalled()
     })
   })
+
+  test.each([
+    { enabled: true, expected: "true" },
+    { enabled: false, expected: "false" },
+  ])(
+    "when initializeRouteFollowerEnabled is $enabled and route is selected, follower should be $expected",
```

Comment:
**nitpick:** the "follower should be $expected" part of the name for these tests seems a little weird to me since that will just expand to (say) "follower should be true." Maybe it should actually say "enabled" / "disabled?"

Reply:
does this work? 6df71681a397677b9ba2b7b7341ef5b90b6b1e95

Reply:
I like it!